sdspi/src/lib.rs

```diff
@@ -69,6 +69,17 @@ pub enum Error {
     WriteError,
 }
 
+/// Must be called between powerup and [SdSpi::init] to ensure the sdcard is properly initialized.
+pub async fn sd_init<SPI, BE>(spi: &mut SPI) -> Result<(), BE>
```

Comment:
We should also take the CS pin here and dessert it (setting it high)

Reply:
Done